content/cli/orch_cli/blueprints.md

```diff
@@ -310,6 +310,53 @@ Blueprints:
 ...
 {{< /highlight >}}
 
+### summary
+
+#### Usage 
+`cfy blueprints summary <target field> [optional sub-field] [OPTIONS]`
+
+Gives a summary of blueprints based on the target field and an optional sub-field.
```

Comment:
I think it's unclear to define a term of summary with that term, I didnt understand what that means until I read the examples

Reply:
Can you think of a clearer way to structure it? I couldn't while I was working through this, but I'll bounce it around in my head.

Reply:
Maybe something like "gives a sum up of elements according to the summary by field"